src/ApparentHorizons/Tags.hpp

```diff
@@ -325,6 +326,23 @@ struct SurfaceIntegral : db::ComputeTag {
   using argument_tags = tmpl::list<AreaElement<Frame>, IntegrandTag,
                                    StrahlkorperTags::Strahlkorper<Frame>>;
 };
+/// Tag representing the surface area of a Strahlkorper
+struct Area : db::SimpleTag {
+  using type = double;
+};
+/// Computes the surface area of a Strahlkorer, \f$A = \oint_S dA\f$ given an
+/// AreaElement \f$dA\f$ and a Strahlkorper \f$S\f$.
+template <typename Frame>
+struct AreaCompute : Area, db::ComputeTag {
+  static std::string name() noexcept { return "AreaCompute"; }
```

Comment:
If the name is the same as the struct name, you can remove this, and it will get defined automatically. I think this is a new feature since these tags were written last summer. Please remove this line specifying name()

Reply:
I'm getting this error when I compile without specifying the name. 

error: static_assert failed due to requirement '!is_compute_item_v<StrahlkorperGr::Tags::AreaCompute<Frame::Inertial> >' "Compute tags must have a name function or a base alias."
  static_assert(not is_compute_item_v<Tag>,

Reply:
It might help to add a line defining a type alias `base`:
```
using base = Area;
```

Reply:
Nice! Thank you, that worked and compiled. 

template <typename Frame>
struct AreaCompute : Area, db::ComputeTag {
  using base = Area;

Reply:
Did the `using base = Area;` end up in the wrong commit or missed by accident? I can't seem to find it

Reply:
I think we had compiler errors we couldn’t figure out, but I might misremember. @MarloMo could you please try adding the using base line to the compute tag (again?) and report any compiler errors, so we can figure out how to proceed?

Reply:
Yes, it was compiling on ocean but I was getting a catch test failure on Github. 

FAILED:
 CHECK( ::db::tag_name<Tag>() == expected_name ) 

with expansion: 
     “Area” == “AreaCompute” 

Reason: A catch test has failed. 
Charm++ fatal error:

A catch test failed. 

@geoffrey4444 @nilsdeppe

Reply:
@MarloMo please update the test so expected_name is "Area", not "AreaCompute"

Reply:
@geoffrey4444 done, looks like everything passed.